CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+Everyone is welcome to contribute to Kaoto. 
+
+We have a Kanban board with good first issues in https://github.com/orgs/KaotoIO/projects/4
+
+There are many ways to contribute to Kaoto:
+
+## Issues
+
+### Submit a new issue
+
+If you have a bug or a suggestion for a new feature, you can create new issues.
+Always **use one of the templates available** and answer as many of the questions as you can.
+
+If you are **submitting a bug**, provide a simple step by step explanation on how to reproduce it and what is the expected outcome.
+
+If you are **submitting a feature**, be ready to follow up and contribute with its development. Features that are proposed but don't have
+funds or developers ready to implement it may be closed due to not enough interest raised. If you can't fund or implement it yourself and 
+you want the feature implemented, you must look for a way to find resources to implement it.
```

Comment:
what if it's a feature we like the idea of? 😆

Reply:
Then we are the resource to implement it. But better if someone else does.